### ISSUE TITLE
Future dependency issue

### ISSUE THREAD

**Issue by svechinsky** [Miserlou/Zappa#1940]
When using Zappa with zenpy there is a dependency conflict.
Zappa requires future==0.16.0 and zenpy requires futuer>=0.17.1

This causes a dependency conflict, I'm sure other packages (not just zenpy might be effected as well)

## Possible Fix
Change the future requirement to:
future>=0.16.0

A similar fix was done for PyYAML:
https://github.com/Miserlou/Zappa/pull/1786

## Steps to Reproduce

Relevant pipenv output:
```
[pipenv.exceptions.ResolutionFailure]:       pipenv.exceptions.ResolutionFailure: ERROR: ERROR: Could not find a version that matches future==0.16.0,>=0.17.1
[pipenv.exceptions.ResolutionFailure]:       Tried: 0.0.1, 0.0.2, 0.0.3, 0.1.0, 0.2.0, 0.3.0, 0.3.1, 0.3.2, 0.3.3, 0.3.4, 0.3.5, 0.4.0, 0.4.1, 0.5.0, 0.5.1, 0.5.2, 0.6.0, 0.7.0, 0.8.0, 0.8.1, 0.8.2, 0.9.0, 0.10.0, 0.10.1, 0.10.2, 0.11.0, 0.11.1, 0.11.2, 0.11.3, 0.11.4, 0.12.0, 0.12.1, 0.12.2, 0.12.3, 0.12.4, 0.13.0, 0.13.1, 0.14.0, 0.14.1, 0.14.2, 0.14.3, 0.15.0, 0.15.1, 0.15.2, 0.16.0, 0.17.0, 0.17.1, 0.18.0
[pipenv.exceptions.ResolutionFailure]: Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
```

**Comment by rudyryk** [comment]
Btw, do we need "future" at all for Python 3.x? Just created #1966 issue.

**Comment by jneves** [comment]
This has been fixed with the merge of #1998
